context/app/api/vitessce.py

```diff
@@ -254,7 +269,7 @@ def _build_assets_url(self, rel_path, uuid):
         else:
             assets_endpoint = MOCK_URL
         base_url = urllib.parse.urljoin(
-            assets_endpoint, str(Path(uuid) / Path(rel_path))
+            assets_endpoint, str(Path(self.uuid) / Path(rel_path))
```

Comment:
```suggestion
            assets_endpoint, f'{self.uuid}/{rel_path}'
```
I think pathlib is great for filesystem paths, when there's a chance you'd do file-systemy things with it, but if you really just want to put a `/` between things, it's fine to do that, I think.